cli/src/commands/xlr/compile.ts

```diff
@@ -6,24 +6,10 @@ import globby from 'globby';
 import logSymbols from 'log-symbols';
 import { TsConverter } from '@player-tools/xlr-converters';
 import type { Manifest } from '@player-tools/xlr';
-import { isNodeExported, isTopLevelNode } from '@player-tools/xlr-utils';
 import chalk from 'chalk';
 import { BaseCommand } from '../../utils/base-command';
-
-const PLAYER_PLUGIN_INTERFACE_NAME = 'ExtendedPlayerPlugin';
-const customPrimitives = [
```

Comment:
Should these extension names live as part of the `Manifest` type?

Reply:
The custom primitives? That might not be a bad idea, it would make them a bit more portable.

Reply:
Opening https://github.com/player-ui/tools/issues/33 to address this outside of this PR